hawc/apps/summary/schemas.py

```diff
@@ -0,0 +1,22 @@
+from pydantic import BaseModel
+
+from . import constants
+
+
+class VisualConfig(BaseModel):
```

Comment:
I mean a schema for the data coming in as a request for a dataset, not the data going out. You don't have to add all fields, just the things like that may be required to get the correct dataset for filtering. In the example above, youd need an assessment which is an int, and a visual type, which is an int too. I'd keep it simple, we can always add more.

Docs on how extra is ignored: https://docs.pydantic.dev/latest/api/config/#pydantic.config.ConfigDict.extra